### PR TITLE
Refactored the physical button-like styling of the buttons to be more controlled

### PR DESCRIPTION
**Pull request by roryaronson**
My last commit was causing elements to behave like buttons that I didn't want. Fixed with a `button-like` class

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -15,7 +15,6 @@
  *= require manual_control
  *= require sequence_builder
  *= require ng-sortable
- *= require help_we_need_a_designer
```
Comment (RickCarlino) [inline app/assets/stylesheets/application.css.scss]: Problem solved.

File: app/views/dashboard/ng-partials/_devices.html.haml [hunk 2]
```diff
@@ -14,29 +14,31 @@
...
+              %label UUID *
               %input{placeholder: "ad698900-2546-11e3-87fb-c560cb0ca47b", type: "text", ng_model: 'form.uuid', required: true}/
             .large-6.columns
-              %label Skynet Security Token *
+              %label Security Token *
               %input{placeholder: "4bbd2jm242dl5wmimbwz4rvlu77m0a4i", type: "text", ng_model: 'form.token', required: true}/
           .row
             .large-12.columns
-              %button.green
+              %button.green.button-like
```
Comment (RickCarlino) [inline app/views/dashboard/ng-partials/_devices.html.haml]: Is this going to change the behavior of foundation buttons?

Comment (roryaronson) [inline app/views/dashboard/ng-partials/_devices.html.haml]: `button-like` makes the button act like a real life button - when it is clicked, it depresses slightly. But we don't want it for every button so I made it its own class.

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (roryaronson) [inline app/assets/stylesheets/application.css.scss]: :+1:

**Pull request merged**